styles.css

```diff
@@ -79,3 +78,168 @@ nav {
   align-items: center;
   color: #505f79;
 }
+
+.work {
+  margin: 25px 0;
+  display: grid;
+  row-gap: 25px;
+  padding: 100px 10px 50px 10px;
+  background: #e5e5e5;
+  justify-items: center;
+}
+
+.card {
+  background: #fff;
+  border-radius: 10px;
+  min-height: 3em;
+  border: 1px solid #c6c6c6;
+  padding: 15px 15px;
+  margin-bottom: 70px;
+  display: grid;
+}
```

Comment:
I think you did a great job here however as part of the requirements of the project you should use flexbox to align the items inside of each card. Kindly changes this, please. :pray: :pray: 
![image](https://user-images.githubusercontent.com/98363075/198831421-ff49d36c-ae9d-4f6f-80de-9e1c35c5641a.png)